image.yaml

```diff
@@ -3,7 +3,7 @@ schema_version: 1
 name: "jboss-eap-7/eap72-openshift"
 description: "Red Hat JBoss Enterprise Application Platform 7.2 OpenShift container image"
 version: "1.0"
-from: "jboss-eap-7-tech-preview/eap72:7.2.0"
+from: "jboss-eap-7/eap72:7.2.0"
```

Comment:
https://docs.engineering.redhat.com/display/CE/Image+naming+and+versioning

"Standalone image naming and versioning"

"RELEASE_MODIFIER (optional) - this can have one of the values: alpha, beta or tech-preview. Used to differentiate between stable and pre-releases."

IMHO it's more intuitive to have a qualifier in there so there are no special rules to remember that are different for standalone vs os images.  Beyond that minor point I don't much care whether this is in there or not. But if it's wrong practice then that doc should say that. The way it is now I read the doc and decided to add it in even though I feared it was wrong practice.

Reply:
Never mind; as you reminded my I used --build-tech-preview.

So, +1 to this change for that reason alone; having the os image refer to build flag magic not visible in the source code == fail.